### ISSUE TITLE
Changing the name of "funky god mode"...

### ISSUE THREAD

**Issue by ardvark71** [Secretchronicles/TSC#441]
Hi all...

As requested here...

http://forum.secretchronicles.de/topics/256

I thought by changing the name of this, it would make the game more family friendly and, perhaps, more agreeable to Christian families/players. :-)

Perhaps a name like "super mode" or something similar to that.

Thanks! :-)

**Comment by datahead8888** [comment]
> I thought by changing the name of this, it would make the game more family friendly and, perhaps, more agreeable to Christian families/players. :-)

Yes, this can be viewed as pagan.  We have to be a bit careful, however - even the presence of magic is viewed as unacceptable in a game - some people view the Lord of the Rings as pagan.  If you think about it, though, all of the Mario games have magic at some level.  It is not possible to account for everything.  We certainly should be paying attention to this feedback, though, since TSC and Super Tux have a strong Christian audience.

I think it should be changed in order to make us distinct from Secret Maryo Chronicles; we need to make our own mark.

My suggestion earlier was "Omega Mode".  There were some other suggestions in the IRC room, but I will let people speak for themselves.

Also, @ardvark71, come join us in IRC chat some time.
Server: http://www.freenode.net
Channel: #secretchronicles

**Comment by ardvark71** [comment]
Hi datahead8888...

Thank you for your thoughts and the invite! :-)

Regards...

**Comment by Luiji** [comment]
If we dedicate ourselves to this level of censorship for Christian families we'll have a lot of difficulty with creative freedom in this game. In addition, if we serve to Christian families, which other denominations should we be concerned about? I don't know enough about every religion to make this game acceptable to everyone. Personally, I wouldn't change anything without direct complaint from an unhappy party.

Additionally, I like the idea of *keeping* funky god mode as a nod to the original codebase.

**Comment by Bugsbane** [comment]
I'm not a Christian, but personally it seems like a small and easy enough change that has little impact on the game itself to warrant concern.

So here are my suggestions:
Funky Allah mode
Funky Ganesha mode
Funky Torah mode
Funky wicca more
Funky Gaia mode
Funky Zeus mode
Funky Thor mode


All kidding aside though, the part before the suggestions*was* serious. We could just call it something simple like "super mode".

**Comment by Bugsbane** [comment]
On and I forgot... "Funky Xenu mode" :P

**Comment by sauer2** [comment]
On 10.07.2015 19:34, Bugsbane wrote:
>
> I'm not a Christian, but personally it seems like a small and easy 
> enough change that has little impact on the game itself to warrant 
> concern.
>
>
In this, that's true.

But in general, I don't even see the problem.

The bunch of funky nutcases that is offended by remotely religious 
sounding terms like "funky god mode" wouldn't probably use a devils 
machine aka computer anyway, much less let their children play games on it.

**Comment by ardvark71** [comment]
Hi sauer2...

A nutcase? That's ok, I've been called a lot worse! :-D

Really for me, it's an irreverent use of the name of God. It's treating something that is Holy and Pure and treating it as if was common or like mud. Kind of like how a lot of people will swear using Jesus's name, using as a curse word. Here's an article that explains this in more detail...

http://www.redlandbaptist.org/sermon/honor-gods-name/

My thought was that seeing God's name used in this manner would turn away potential Christian players quicker than it would take a 6 year old to open a candy wrapper, hence why I posted this thread. Yes, many Christians like myself do use computers, even work on them. Like any human tool, they can be used for good or bad. ;-)

Best wishes...

**Comment by JoshDreamland** [comment]
Personally, I'd be more inclined to see the name changed if not for the sheer number of games and engines that use the phrase "god mode." The subtle distinction is the capitalization of the word "God." The word appears capitalized in biblical text as a proper noun, along with "Lord" and "Father," and even pronouns which are made proper nouns, such as "He"/"Him." In English text, "god" doesn't refer to the biblical God, but to any kind of deity or higher being, eg, the Greek gods. Keeping it lower-case disambiguates this, which is why "Funky god mode" is a completely legitimate name. Not to mention, this would not be the only children's game to contain a non-biblical use of "god." Take Scribblenauts, for example: a children's game in which gods can be spawned in any number, and even destroyed. (Note that in this particular game, priests are somehow invulnerable. No further comment from me, there.)

**Comment by sauer2** [comment]
No offense, I still don't think it is a worse problem than bowing to the 
whim of religious groups. Even if they are not a small minority. 
Especially then, because why draw a line. Popularity doesn't mean 
legitimation.

I don't care so much for this particular case, but remember: If you go 
that way, next thing that could happen is have Scientologists/church of 
whatever to show up and claim offense on one of the story characters name.
Next up: Symbols and key actions of the story that could offend someone.

On the other hand, since I'm only ... uhm semi-involved in this project, 
you all better duke this matter out on IRC.

**Comment by kirbyfan64** [comment]
I think the difference is that this is a really easy fix...

**Comment by sauer2** [comment]
Nonetheless: You all might want to put up a policy if such issues arise 
in future.

**Comment by datahead8888** [comment]
I had suggested changing it in order to make our own mark, but Luiji has brought up a good point that SMC is still part of our heritage.  There is no right or wrong answer here.

As JoshDreamland said, I've also seen "god mode" in another game before, which means FluXy or whoever coded this feature was just following conventions.

sauer2, no, we cannot make special exceptions for all sorts of requests by specific groups, since it's not possible to account for everything, unless something is blatantly offensive to a group.  It was suggested by someone a while back that we abstain from magic for similar reasons, which could severely limit our story depending on what direction we take.

We might consider taking a vote and/or adding this to the next IRC meeting agenda for further discussion, in order to give fair consideration to all points of view.

**Comment by Luiji** [comment]
@datahead8888 It's actually a common name in gaming. [1] One of the reasons I like it is somebody might guess it randomly. I'd even like to make it work in the pause menu, since that's where most people try cheat codes.

[1] http://www.urbandictionary.com/define.php?term=god+mode&defid=836842 (def. 1)

**Comment by Bugsbane** [comment]
Minor change. No impact on gameplay. 10 seconds to implement. Very important to a certain demographic of users. I just don't see why this is an issue of debate at all.

The only argument's I've seen against this are that it would be kow-towing to religious pressure, or that it's somehow dishonoring the game's heritage.

When the change doesn't impact anything meaningful to the gameplay, or nature of the game, then fighting it just so as to not let "religious groups" have their way, seems to be fighting just for the sake of fighting to me. As to the game's heritage, that seems pretty laughable given the many, many changes we're making that are 100 times larger.

As to the "if we give them an inch, they'll demand a mile" argument, it seems to me that the appropriate response is to resist to the level that the suggested changes are damaging to the game.

If some group wants to make unreasonable demands that nothing magical be in the game, I would fight that to the very end, as it significantly damages the possibilities in the game for no real benefit. In this case it's the reverse. We have a change that doesn't do any damage to the game, is important to part of the player demographic, and as far as I can see, we're blocking it for no real gain beyond a "just don't let 'them' win anything" mentality.

Even as a non Christian - or member of any religion - I find the arguments against even being willing to discuss other options to be unnecessarily antagonistic. It's not like anyone is proposing we call this the "Jesus Christ is our Lord and saviour mode". Heck, we could even just call it something like "Funky mode", and still be referencing our games heritage while keeping our theistic players happy, too. Is that really so offensive to the more atheistic members? In all honesty, I would have thought the removal of the word "god" would have pleased both sides here (if we're to reduce this to a polar argument).

**Comment by Luiji** [comment]
I find it inane to try and remove the usage of the word god from everywhere you can. I think @Bugsbane is right that this is a minor change but I think he's wrong that this shouldn't be debated. Changing every tiny little thing for everyone is, in my opinion, bad policy.

At the same time, I'd feel hypocritical trying to shove my opinion down everybody else's throat and fighting for such a minor thing to change. If anybody put this up to a vote I'd put myself down for no-opinion, as I don't care nearly enough to push the issue. We have more interesting things going on in this project.

**Comment by Bugsbane** [comment]
Personally I'd go for F U N K and Y. :-)

**Comment by Bugsbane** [comment]
By the way, I'm not arguing that anything is beyond debate. I'm just not seeing any great benefit to NOT changing it, while for some of our users there is a great benefit to changing it. Also I disagree that making this one small, simple change is equivalent to "Changing every tiny little thing for everyone". The debate has now taken up probably 100 times the time that would have been needed to create the patch.

**Comment by datahead8888** [comment]
>  find it inane to try and remove the usage of the word god from everywhere you can. I think @Bugsbane is right that this is a minor change but I think he's wrong that this shouldn't be debated. Changing every tiny little thing for everyone is, in my opinion, bad policy.

We had someone complain again and again until we enforced certain coding conventions, which are now called upon in every code review.  Thus this issue can actually be larger than just a question as to whether or not we yield to religious groups.  I'm not saying Bugsbane or advark have done anything wrong here; I'm just saying it's a policy consideration for other issues in the future.  

I don't have a problem with changing if it's not just for the sake of removing "god".  Also, not all Christians find the usage of of funky god mode offensive.  I wouldn't want to push my view either; I had suggested removing god mode further above (though for different reasons).

If we can't come to agreement, as Lujiji said, we will have to take a vote.  We could also add it to the next IRC meeting agenda, but that meeting probably wont' happen for at least a few months.

**Comment by Bugsbane** [comment]
Anyway, at this point I feel like I've made my points, and I feel like we've probably heard all of the counterarguments. I'm prepared to listen to anything anyone wants to add, but I agree that it's probably down to a vote at this point.

**Comment by Quintus** [comment]
Who of you all had the idea to start a religious argument in the tracker while I am doing my exams so I can’t intervene and stop it?

This issue’s discussion has entered the highly complicated area of social issues that do not have a technical relation. If we aren’t careful enough, we’re going to end up like this: https://github.com/opal/opal/issues/941 — nobody of us wants that.

As has been pointed out, it is impossible to make a game that will not hurt one or others feelings in some way. Please be aware that nobody is forced to play this game, and if he dislikes our use of the term “god mode” he can freely decide to cease use of TSC.

It is void to discuss such a social issue. As long as no code or asset is added to the game with the _intention_ to offend somebody (which the god mode for sure did not have), either way (keeping or changing the term) is fine. Thus it is best to just vote on it, and do so again if any other religious or social complaints arrive.

Please do not start a vote in the next days while I’m still doing my exams. I will take care of that when I have finished them.

Valete,
Quintus

**Comment by Luiji** [comment]
@Quintus You have no idea how many times I've rewritten every post I've made in an attempt to avoid this becoming a religious argument instead of just a policy argument. ;) This topic has taken way to many brain cells just out of total fear of what happens if I express myself the wrong way. Also damn that Opal topic... *shudders*

@Bugsbane My point is not that changing this is equivalent to "Changing every little thing etc.", it's that if we just change it without any debate we would've been implicitly setting that policy. I said so because I believed that you're earlier point was that this particular topic wasn't worth debate because it required little effort to implement. My apologies if I interpreted you wrong.

Also, @Bugsbane, holding down F-U-N-K-Y simultaneously is insanely hard on a QWERTY keyboard. Maybe we should just keep the same key binding but suggest it's actually "DOG"? Since they're all pressed in unison, it can technically mean *any* anagram.

**Comment by kirbyfan64** [comment]
@Luigi It can be called...

...

TOP DOG MODE!

**Comment by datahead8888** [comment]
@Quintus - some discussion is needed any time we are going to take a vote so that we can each decide how we will vote.  You are correct that it's time to just take a vote here, though.

**Comment by sydneyjd** [comment]
Im sorry for taking so long to answer, but here is my view.
Im personally fine with "Funky God mode" or "GOD" mode, mainly because its short, and for Christians(me for one) its impossible to get away from people refering to gods as invincible, so in a way GOD mode makes sense.
But then again, i do know what angle @ardvark71 is coming from, and he does have a good point. 
If we had to change it i would go with DOG mode also, but a dog itself(at least here) usually refers to something lowly and not powerful.
How about we do "DOG" for the keybinding and call it "Underdog Mode". :stuck_out_tongue:

**Comment by Luiji** [comment]
@sydneyjd I'm cool with Underdog Mode. :)

**Comment by Quintus** [comment]
To bring this insane issue to a conclusion, is everybody fine with me calling for a formal team vote on it? Everybody can then vote whatever he thinks is best, and we’ll just go with the majority result.

To make the vote a useful one, please give useful, nonsatirical alternative names that I can include in the voting statement so that we don’t have to go voting again when the decision falls in favour of renaming. We’ll just do it all in once.

Valete,
Quintus

**Comment by sauer2** [comment]
Am 07.10.2015 um 14:52 schrieb Marvin Gülker:
>
> To bring this insane issue to a conclusion, is everybody fine with me 
> calling for a formal team vote on it?
>
Sure.

**Comment by Bugsbane** [comment]
Bring it.

**Comment by sydneyjd** [comment]
Im still stuck on underdog mode. ;p Would we have copywrite issues,etc if we used it?

**Comment by Quintus** [comment]
> Im still stuck on underdog mode. ;p Would we have copywrite issues,etc if we used it?

None I am aware of.

So I think the following choices are possible:

* Funky God Mode
* Funky god mode
* God Mode
* God mode
* Dog Mode
* Underdog Mode

If no more suggestions are made until the weekend, I’ll issue a vote. Until then, please feel free to make further naming suggestions.

Vale,
Quintus

**Comment by sydneyjd** [comment]
Another thought is Invincible Mode.

**Comment by datahead8888** [comment]
Omega Mode is another one, if we change the name

**Comment by ardvark71** [comment]
I like sydneyjd's suggestion, "Invincible Mode." :-)

**Comment by Bugsbane** [comment]
I like both invincible move and omega mode. If I had to choose, is choose the former.

Not such a big fan of dog mode and similar, because really, what do dogs have to do with what the mode actually does? Nothing. It's really not clear at all.

**Comment by sydneyjd** [comment]
Maybe its more of an American term, but im not sure. Back in the 1960 a
Show called 'UnderDog' Was made about a superhero dog.

**Comment by Quintus** [comment]
> Maybe its more of an American term, but im not sure. Back in the 1960 a Show called 'UnderDog' Was made about a superhero dog.

That’d made more for a subtile reminiscence than for a copyright infringement :-)

Vale,
Quintus

**Comment by Quintus** [comment]
> Not such a big fan of dog mode and similar, because really, what do dogs have to do with what the mode actually does?

Read “dog” reversed. :-)

Vale,
Quintus

**Comment by datahead8888** [comment]
> That’d made more for a subtile reminiscence than for a copyright infringement :-)

For Under Dog mode - I don't really think we should name our invincible mode off of a specific super hero.

As for dog mode, most people will be confused by this and not realized it's a reversal of the letters in "god".

**Comment by Quintus** [comment]
> As for dog mode, most people will be confused by this and not realized it's a reversal of the letters in "god".

Cat mode

cat mode

cat(1) mode

**Comment by Quintus** [comment]
~~~~~~
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

                A N N O U N C E M E N T   O F   V O T E


2nd vote                                           October 11th, 2015

This is a formal vote announcement as per § 5 of the Voting Rules
(VotR), which are available online[1]. A vote that is accepted by the
team results in a final decision on a topic and determines how to
proceed.

Allowed to vote is any team member, which as per § 1 VotR is everybody
who is publically listed as a member of the “Secretchronicles”
organisation on GitHub.

Proponent
°°°°°°°°°

The vote was proposed by me, Quintus <quintus@quintilianus.eu>.

Vote Subject
°°°°°°°°°°°°

There was a discussion spawned in the issue tracker (ticket #441[2])
about whether or not to rename the special mode that makes Alex
invincible from “Funky God Mode” to something different. The main
argument was that the term might be perceived as an unnecessary and
maybe even blasphemic use of the name of God for highly religious
Christians. There has been a good number of pros and cons being
exchanged on the tracker, which I advise everyone to carefully read
before making a decision.

As it appears to be impossible to find a compromise (one can’t really
change the name only partly) and there are people arguing for both
options, I determined a vote was necessary to clear the matter up.

To ease things, this vote not only decides about whether to rename
current God Mode, but also about what to rename it to exactly to prevent
nececcisity of another voting procedure. By voting for option 1 below,
you indicate you want to keep the current naming, voting for any other
option is a vote against the current naming and in favour of another
name (the one you vote for).

Vote Options
°°°°°°°°°°°°

Option 1: Funky God Mode (i.e. keep the current naming)
Option 2: Funky god mode (“God” not capitalised)
Option 3: God Mode
Option 4: Dog Mode
Option 5: Invincible Mode
Option 6: Omega Mode
Option 7: Underdog Mode

Voting Period
°°°°°°°°°°°°°

The voting period starts on 2015-10-25 00:00 UTC, and it ends on
2015-11-01 23:59 UTC.

Procedure
°°°°°°°°°

On 2015-10-25 I will post the public Call for Votes (CfV) that
officially opens the voting period. After that you may vote by either
emailing your chosen option to the mailinglist, posting it to the forum
into the topic containing this announcement, or posting it on the
tracker in the ticket where this announcement was published. Any vote
given before that CfV was posted is ignored. The timespan of two weeks
between this announcement and the CfV is intended to give you time to
decide and prepare, to allow you making last efforts in convincing
others from your opinion, and to decide whether you request a covert
vote (see below).

If no single option receives the most votes (but multiple ones win), a
second vote is held with the set of options reduced to those options
that received the most votes.

If option 1 wins, the current name of “Funky God Mode” will not be
changed now or in the future. Any other option means changing the name
to whatever got the most votes.

Covert vote
°°°°°°°°°°°

Each team member has the right to request a covert vote. A covert vote
means that a technical procedure is employed that prevents everyone from
seeing how anyone else has voted. It is intended to prevent conflicts
within the team or between users and specific team members, and you
should only request this if you really think this is required. More
openness is better for the project.

If the request is made, it must be honoured. I will announce the
technical procedure in that case.

The request must be made before the Call for Votes has been posted, any
further requests for covert votes are ignored.

Quintus
TSC project lead.

[1]: https://github.com/Secretchronicles/documents/raw/master/votingrules/votingrules-2015-08-15.pdf
[2]: https://github.com/Secretchronicles/TSC/issues/441
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2

iQEcBAEBCAAGBQJWGgaZAAoJEPHYeZ+8yLxPSVIIALPSnR5LUXtNFmxWxQldSTuL
OAhPTATXPITCZtkhjm/W5YcexdMlX1zivJtebPHu0LVhUXkDZtC+OyxQ02C6/6D1
Qn9E+mZxSuczCtGqtV6fGb6M3K5cXXheH/bezXQ4jP6YSl6T+fNn+ubFgl0KOTpk
7zMYJyo2Oeu86XPLnvOh+Tqgfo3o63UkPEbG4lVt3ZM1Gz4SB/L7YhwQcq9NKoyI
U17zgn9aETD7jnOpSB/zR1s8cKZ+QTpeSsQk2ATVvGb+JuAEvz8G/2VdjCoKo+4f
/uFeZEN3UMq9mFNp+H/mQCnjoztkXssmPwSc8J1onAKleLMGcSA4TiGvvESRF3Y=
=zFDT
-----END PGP SIGNATURE-----

~~~~~~

**Comment by Quintus** [comment]
The problem I have with “Invincible Mode” is that this might be confused with the effect the lemon powerup has, which basically is an “invincible effect” also. I’d thus rather go with “Omega Mode” I think, which I think is a pretty cool name.

Vale,
Quintus

**Comment by Bugsbane** [comment]
> this might be confused with the effect the lemon powerup has, which basically is an “invincible effect” also. I’d thus rather go with “Omega Mode”

Ok. I'm convinced. My vote is for Omega Mode.

**Comment by Quintus** [comment]
Beware :-)

> The voting period starts on 2015-10-25 00:00 UTC, and it ends on
> 2015-11-01 23:59 UTC.
> [...]
> On 2015-10-25 I will post the public Call for Votes (CfV) that
> officially opens the voting period. [...] Any vote
> given before that CfV was posted is ignored.

:-)

**Comment by Bugsbane** [comment]
Ok... My vote *will be* for Omega Mode on the 25'th ;)

**Comment by datahead8888** [comment]
> My vote will be for Omega Mode on the 25'th

At least I know my suggestion (if we rename it) was popular :P

**Comment by Bugsbane** [comment]
Maybe we should cash in on your popularity, and change it to "datahead mode"... ;P

**Comment by datahead8888** [comment]
There's nothing wrong with being happy if a suggestion is considered...

**Comment by ardvark71** [comment]
Who all is allowed to vote on this? I didn't really understand this from the notice.

**Comment by Bugsbane** [comment]
> There's nothing wrong with being happy if a suggestion is considered...

Absolutely! I didn't intend to imply otherwise. Apologies if it came across that way.

**Comment by Quintus** [comment]
> Who all is allowed to vote on this? I didn't really understand this from the notice.

Quote from the notice:

> Allowed to vote is any team member, which as per § 1 VotR is everybody
> who is publically listed as a member of the “Secretchronicles”
> organisation on GitHub.

The document is available [here](https://github.com/Secretchronicles/documents/blob/master/votingrules/votingrules-2015-08-15.pdf), and the members of the Secretchronicles GitHub organisation are listed [here on GitHub](https://github.com/orgs/Secretchronicles/people). The reasoning behind this is to have a definitive set of people that can vote which can easily determined by looking at one single piece of information so that there is no dispute about who can and who cannot vote. Also, everybody is free to earn team membership by contribution to the game and thereby gain a right to vote.

Vale,
Quintus

**Comment by ardvark71** [comment]
Hi Quintus...

Thank you for clarifying that. :-)

Regards...

**Comment by sydneyjd** [comment]
I like with Omega mode, and it fits, because the lemon *is* invincible mode. Do we really have to do a vote?

**Comment by datahead8888** [comment]
As I said, I don't think god mode had to be changed, but I am fine with just going ahead and changing it to omega mode here.

**Comment by Quintus** [comment]
> I like with Omega mode, and it fits, because the lemon is invincible mode. Do we really have to do a vote?

I still want to keep the current naming, and you can’t know how the people are going to vote that are not yet active in this issue. So the answer is yes. Also, after the vote announcement is out, you can’t stop the voting process anymore without the proponent agreeing, which is me :-P

Vale,
Quintus

**Comment by datahead8888** [comment]
I seem to remember there being discussion about having two votes: one on whether or not to change god mode and another one what name to use if we change it.  Did we decide to only hold one vote for all decisions altogether?

I saw a reference to "kid" mode in the SFML port code.  What is planned for this?

**Comment by Quintus** [comment]
datahead8888 <notifications@github.com> writes:

> I seem to remember there being discussion about having two votes: one
> on whether or not to change god mode and another one what name to use
> if we change it.  Did we decide to only hold one vote for all
> decisions altogether?

Quote from the announcement:

> To ease things, this vote not only decides about whether to rename
> current God Mode, but also about what to rename it to exactly to prevent
> nececcisity of another voting procedure.

Reasoning is that a formal voting procedure is a heavy operation. Each
vote needs at least two weeks of preparation time (between vote
announcement and CfV). If you wanted to do two votes you would thus at
least need four weeks.

> I saw a reference to "kid" mode in the SFML port code.  What is
> planned for this?

This is a rather unknown second cheat in TSC, and has nothing to do with
the SFML port (it was copied over). It is already there (it was
inherited from SMC, where it already existed). Try pressing keys K, I, D
the same way you press G, O, D for (current) god mode.

Vale,
Quintus

**Comment by datahead8888** [comment]
> Reasoning is that a formal voting procedure is a heavy operation. Each vote needs at least two weeks of preparation time (between vote announcement and CfV). If you wanted to do two votes you would thus at least need four weeks.

I think it would be reasonable to announce two votes at once and only wait two weeks for them together.  However, in this case, adding the second vote would require us to wait two weeks from the point at which we changed the voting for this issue, which would drag this out.

**Comment by Quintus** [comment]
Of course it is possible to have multiple votes running at once, but in this case the second one would depend on the result of the first one. So I thought merging them was the simplest approach.

Vale,
Quintus

**Comment by datahead8888** [comment]
I misunderstood you before.  Yes, it is simpler this way, but two votes can allow people to express an opinion both ways in such a case.  It is too late to change this vote now, however.  I failed to notice the change myself anyways.

**Comment by Quintus** [comment]
~~~~
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256


                      C A L L   F O R   V O T E S


2nd vote                                          October 25th, 2015


Hi everyone,

this is a formal Call for Votes (CfV) as per § 9 VotR. It officially
opens the voting period for the vote described below, that is, you are
now free to exercise your right to vote. A vote that is accepted by the
team results in a final decision on a topic and determines how to
proceed, so please take care before you participate.

Allowed to vote is any team member, which as per § 1 VotR is everybody
who is publically listed as a member of the “Secretchronicles”
organisation on GitHub. These are the following eight people:

- - brianvanderburg2, aka simpletoon
- - DarkAceZ
- - datahead8888
- - Luiji
- - Quintus
- - sauer2
- - sydneyjd
- - xet7

Proponent
°°°°°°°°°

The vote was proposed by me, Quintus <quintus@quintilianus.eu>.

Vote Subject
°°°°°°°°°°°°

There was a discussion spawned in the issue tracker (ticket #441[2])
about whether or not to rename the special mode that makes Alex
invincible from “Funky God Mode” to something different. The main
argument was that the term might be perceived as an unnecessary and
maybe even blasphemic use of the name of God for highly religious
Christians. There has been a good number of pros and cons being
exchanged on the tracker, which I advise everyone to carefully read
before making a decision.

As it appears to be impossible to find a compromise (one can’t really
change the name only partly) and there are people arguing for both
options, I determined a vote was necessary to clear the matter up.

To ease things, this vote not only decides about whether to rename
current God Mode, but also about what to rename it to exactly to prevent
nececcisity of another voting procedure. By voting for option 1 below,
you indicate you want to keep the current naming, voting for any other
option is a vote against the current naming and in favour of another
name (the one you vote for).

Vote Options
°°°°°°°°°°°°

Option 1: Funky God Mode (i.e. keep the current naming)
Option 2: Funky god mode (“God” not capitalised)
Option 3: God Mode
Option 4: Dog Mode
Option 5: Invincible Mode
Option 6: Omega Mode
Option 7: Underdog Mode

Voting Period
°°°°°°°°°°°°°

The voting period has started now. It ends on 2015-11-01 23:59 UTC.

Procedure
°°°°°°°°°

As nobody requested a covert vote, this vote will be held as an open
vote. Please vote by sending an email indicating the option you voted
for to the mailinglist, make an according post to the forum topic
corresponding to this message, or post on the tracker in the ticket
where the vote announcement was published as well. Everybody has only
one vote. If you have a GPG keypair, please use it to sign the message
(this is optional, but recommended).

If no single option receives the most votes (but multiple ones win), a
second vote is held with the set of options reduced to those options
that received the most votes.

If option 1 wins, the current name of “Funky God Mode” will not be
changed now or in the future. Any other option means changing the name
to whatever got the most votes.

Quintus
TSC project lead.

- -- 
#!/sbin/quintus
Blog: http://www.guelkerdev.de

GnuPG key: F1D8799FBCC8BC4F
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2

iQEcBAEBCAAGBQJWLTAwAAoJEPHYeZ+8yLxP9g4H/1KIIDlV282Cc5ztXhZsK27C
iltdEQjT8te7DNUf7meHnduoHcUawSKkKFAQX1s/VfYCcrB0UWNDjozaVEqDTkRo
h9DxryPbU+aXxGXtipR9boVximRNl4vglJ40iDwY3MEmglZQ9AZ7qwAGW+nDAc3j
Gw1PNQENb7Uick8HAKPuAEL8QqWK4y650f5KoQkiZLL7BXLvsn4R/ofeTsO2MZTu
iwWW6XrqzuApVYJlb5nrerA5mivpgyiOXjyeNxCwreqUPPxDvMHZzYsK5CtPp/45
ZE/NB9zCA0MAiBpeEEsMJBfQD8cnGeJdRhk5Cu6lmh/nc6D1fidGm9DvgWZh1XQ=
=q5Xv
-----END PGP SIGNATURE-----
~~~~

**Comment by sauer2** [comment]
Option 6

**Comment by Bugsbane** [comment]
Just to reiterate, my vote is for option 6 "Omega Mode" (btw, I'm listed as part of the organization here on GitHub, but my name isn't in your comment above @Quintus )

**Comment by Quintus** [comment]
Bugsbane <notifications@github.com> writes:

> Just to reiterate, my vote is for option 6 "Omega Mode" (btw, I'm
> listed as part of the organization here on GitHub, but my name isn't
> in your comment above @Quintus )

It appears your membership is not set to be “public”, i.e. it can only
be seen by people who are members of the organisation themselves. When I
made the list I was not logged in into GitHub, so I didn’t see it. It’d
be a good idea to publicise it then (go to the organisation page and
change your visibility setting to “public”). It’s not a problem that you
aren’t on my list as long as you are on the organisation page :-)

Vale,
Quintus

-- 
#!/sbin/quintus
Blog: http://www.guelkerdev.de

GnuPG key: F1D8799FBCC8BC4F

**Comment by datahead8888** [comment]
> It appears your membership is not set to be “public”

Does Arik Burns (@Aakburns) have the same status?  I seem to remember you saying he was not public either.  We could send him an email both about this vote and about the github public issue.  We also might invite him to join the mailing list if he is not already a member.

**Comment by Quintus** [comment]
> Does Arik Burns (@Aakburns) have the same status?

Just log out of GitHub and view [the organisation page](https://github.com/orgs/Secretchronicles/people). Then log in and view the same page. You’ll immediately notice who is not set as public.

Vale,
Quintus

**Comment by sydneyjd** [comment]
I vote :
Option 6: Omega Mode

**Comment by Quintus** [comment]
For those who didn’t notice: Voting happens in parallel also [on the mailinglist](http://lists.secretchronicles.de/tsc-devel/2015/10/0000075.html). I just say this so you aren’t surprised if there are more votes in the final results than you might expect from this ticket’s discussion.

Valete,
Quintus

**Comment by DarkAceZ** [comment]
Option 6 sounds the best.

**Comment by Quintus** [comment]
~~~~~~~~~~
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256


              D E C L A R A T I O N   O F   R E S U L T S

2nd vote                                          November 3rd, 2015

Hi everyone,

this is the formal declaration of results on the 2nd vote of the TSC
team. It contains the official number of voting ballots counted, their
chosen options, and it states the outcome of the vote.

History
°°°°°°°

- - 2015-10-11: Vote announcement
  http://lists.secretchronicles.de/tsc-devel/2015/10/0000023.html
  <87si5hsxnu.fsf@atlantis.cable.internal.west-ik.de>

- - 2015-10-25: Call for Votes
  http://lists.secretchronicles.de/tsc-devel/2015/10/0000075.html
  <8737wyzqhh.fsf@hades.cable.internal.west-ik.de>

- - 2015-11-03: Declaration of results
  (this message)

Proponent
°°°°°°°°°

The vote was proposed by me, Quintus <quintus@quintilianus.eu>.

Vote Subject
°°°°°°°°°°°°

There was a discussion spawned in the issue tracker (ticket #441[1])
about whether or not to rename the special mode that makes Alex
invincible from “Funky God Mode” to something different. The main
argument was that the term might be perceived as an unnecessary and
maybe even blasphemic use of the name of God for highly religious
Christians. There has been a good number of pros and cons being
exchanged on the tracker, which I advise everyone to carefully read
before making a decision.

As it appears to be impossible to find a compromise (one can’t really
change the name only partly) and there are people arguing for both
options, I determined a vote was necessary to clear the matter up.

To ease things, this vote not only decides about whether to rename
current God Mode, but also about what to rename it to exactly to prevent
nececcisity of another voting procedure. By voting for option 1 below,
you indicate you want to keep the current naming, voting for any other
option is a vote against the current naming and in favour of another
name (the one you vote for).

Votes
°°°°°

- From 9 people allowed to vote (8 mentioned in the announcement message
plus Bugsbane who I assure you is in the team but still has not
publicised his membership, but has pointed that out on the tracker[2]) 7
did exercise their right to vote, which gives a participation of
approx. 77.78%.

These are the definitive results:

| # | Option name                                   | Votes | Percent  |
|---+-----------------------------------------------+-------+----------|
| 1 | Funky God Mode (i.e. keep the current naming) |     2 | ~28.57 % |
| 2 | Funky god mode (“God” not capitalised)        |     0 |   0.00 % |
| 3 | God Mode                                      |     0 |   0.00 % |
| 4 | Dog Mode                                      |     0 |   0.00 % |
| 5 | Invincible Mode                               |     1 | ~14.29 % |
| 6 | Omega Mode                                    |     4 | ~57.14 % |
| 7 | Underdog Mode                                 |     0 |   0.00 % |
|---+-----------------------------------------------+-------+----------|
| - | Total                                         |     7 | 100.00 % |

Effect of the Vote
°°°°°°°°°°°°°°°°°°

Option #6 has been ACCEPTED by the majority of the team. It is a binding
decision the team follows from now on, all circumstances being equal.

With regard to the vote subject this means that the current “Funky God
Mode” is going to be renamed to “Omega Mode”.

Right to Complain
°°°°°°°°°°°°°°°°°

Every team member has the right to file a complaint against this formal
declaration. The complaint must be filed within 14 days from now on
(timestamp of this email) with either the project lead or with the
project assistant lead. The project lead and assistant lead must file
the complaint with one another; if both are involved, a volunteer must
be seeked for.

The complaint has to contain an explanation as to why the voting rules
have been violated. The person the complaint was filed with will then
check the procedure against the requirements of the voting rules
document, and if he finds a violation, he can declare the vote failed,
which allows repetition of the vote in question.

Other
°°°°°

I am going to open an issue on the tracker for keeping track of setting
this vote’s results into effect.

Quintus
TSC project lead.

[1]: https://github.com/Secretchronicles/TSC/issues/441
[2]: https://github.com/Secretchronicles/TSC/issues/441#issuecomment-151079395

- -- 
#!/sbin/quintus
Blog: http://www.guelkerdev.de

GnuPG key: F1D8799FBCC8BC4F
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2

iQEcBAEBCAAGBQJWOQ3nAAoJEPHYeZ+8yLxPSdwH+wfUaPfpK4apTJE0G0wPInHU
tvk8j2+VcSl6XX33yTEZrIV3Um6gDqjodwn0FLQgDDqwdHhk4gfOKesGFMWQx54b
7dtdu8SztT1IkkcLGIw6WnxL+E15ctW/IjVg+4CuKYjB+8TtKk6wbCmIDsnlOosw
AdzBl3y8TqoW0X4xFooRaQ6Gw5Rm5idaeyH9k/eLRJxKBzrf3eu5puW9r55h8lfd
rWh/wFTLxIXSlTLG3zVG9sDaRgGqnSPvjAyr8Uj4V2MQzm2VCAt5drd7LBu0HeNE
l/BEFMIqZOjMlAo+fKo/jUHirEpvPUNdA6GKFf7mBF+MWfCGsllOL1tr+Df4kVc=
=HK1l
-----END PGP SIGNATURE-----
~~~~~~~~~~

**Comment by kirbyfan64** [comment]
Short version:

Omega Mode is the new name.

**Comment by datahead8888** [comment]
> Omega Mode is the new name.

Once someone codes the change and commits it to devel.  It probably would be best to include this in the next release given that we just voted on it.

**Comment by Quintus** [comment]
Superseded by #481.

Valete,
Quintus
